### DISCUSSION TITLE
Pause backups while no players are online

### DISCUSSION THREAD

**Discussion by dghodgson** [mbround18/valheim-docker#297] [Ideas]
Considering that the game state pauses while no players are online, backing up the world when nobody is there is wasteful. 

Additionally you may not be interested in the last N hours/days of backups (where nobody was online), but older backups where changes were actually taking place on the world would still be useful (but may be deleted due to age, or space issues).

Would be awesome to have the backup system check if players are online before running backups.

**Comment by mbround18** [comment]
This would be an excellent addition! I am currently working on the status feature right now and will take a look at potentially adding this <3

**Reply by mbround18** [reply]
I was able to add this to #298  :) with a new  env flag called `AUTO_BACKUP_PAUSE_WITH_NO_PLAYERS`

**Reply by TheDakk** [reply]
Did this go live yet?

**Reply by mbround18** [reply]
Nada, It is still in review since we had to change the repo structure. I can push a dev image up if you are interested?
